redirection_map

```diff
@@ -26,3 +26,12 @@
 /components/templating /components/templating/introduction
 /cookbook/upgrading /cookbook/upgrade/index
 /cookbook/security/voters_data_permission /cookbook/security/voters
+/create_framework/http_foundation /create_framework/http-foundation
+/create_framework/front_controller /create_framework/front-controller
+/create_framework/http_kernel_controller_resolver /create_framework/http-kernel-controller-resolver
+/create_framework/separation_of_concerns /create_framework/separation-of-concerns
+/create_framework/unit_testing /create_framework/unit-testing
+/create_framework/event_dispatcher /create_framework/event-dispatcher
+/create_framework/http_kernel_httpkernelinterface /create_framework/http-kernel-httpkernelinterface
+/create_framework/http_kernel_httpkernel_class /create_framework/http-kernel-httpkernel-class
+/create_framework/dependency_injection /create_framework/dependency-injection
```

Comment:
should we add some tests, which urls are callable?

(this comment is not directly related to this PR, sorry)